### ISSUE TITLE
Clone is excessively large

### ISSUE THREAD

**Issue by the-destro** [bonfire-networks/bonfire-app#467]
Hello. The repo is pretty large on clone(>400MB) but does not map to a current file with in the repo

**Comment by mayel** [comment]
Good point @the-destro! The history contains a lot of stuff that has been extracted to separate repos. Will delete and start fresh soonish.

**Comment by ivanminutillo** [comment]
Most of them are content that should have been in .gitignore i guess?

<img width="1021" alt="Screenshot 2024-07-16 alle 12 02 39" src="https://github.com/user-attachments/assets/357cb252-652f-4c48-852f-be648654e5df">

we can use [git-filter](https://github.com/newren/git-filter-repo) to remove them without start fresh?

**Comment by mayel** [comment]
it's also because this used to be a monorepo before most contents were moved to other repos

**Comment by mayel** [comment]
Done, this repo is now light and fresh. Old repo with full history is archived at https://github.com/bonfire-networks/bonfire-app-archive
